pkg/plugins/policies/meshtimeout/plugin/v1alpha1/plugin_test.go

```diff
@@ -460,6 +460,10 @@ var _ = Describe("MeshTimeout", func() {
 					}},
 				},
 			},
+			RuntimeExtensions: map[string]interface{}{},
+		}
+		for n, p := range core_plugins.Plugins().ProxyPlugins() {
```

Comment:
should we run tese plugins in test? In my opinion we should explicitely provide `RuntimeExtension` in test, since we are not testing proxy plugin in this test

Reply:
Yes that's rigth!

Reply:
Ok it was actually hard to redo it simply so I've only loaded the one gateway plugin instead of all of them